### ISSUE TITLE
DialogHost closing strategy from C#

### ISSUE THREAD

**Issue by montex12** [mysteryx93/HanumanInstitute.MvvmDialogs#12]
Hello,

I'm struggling with `MvvmDialogs.DialogHost` to close the shown dialog from C# and not from XAML.

The example provided shows how to close it using XAML `(AskTextBoxView.axaml)`:
```XML
<Button Content="Ok" Command="{Binding RelativeSource={RelativeSource FindAncestor, AncestorType=dialogHost:DialogHost}, Path=CloseDialogCommand}" CommandParameter="{Binding Text}" />"
```

I have tried to bind a `ICommand` in the ViewModel and close it using `_dialogService.Close(this);`  passing the dialog ViewModel but it does nothing
also `_dialogService.Close(OwnerWindow);` passing the Window that is presenting the dialog content but in this case the whole application terminates.

Consulting the DialogHost documentation it mention to close it using `DialogSession.Close` but I don't know how to obtain it since it's not exposed in MvvmDialogs.

Edit:
I tried also implementing `ICloseable` interface and raising a `RequestClose` event but it doesn't anything.

Thanks

**Comment by mysteryx93** [comment]
There are 2 parts to MvvmDialogs: dialogs, and framework dialogs.

`ICloseable` design was implemented for standard dialogs shown with `Show` or `ShowDialogAsync`. It doesn't apply to framework or custom dialogs like message boxes or DialogHost.

In the sample, it asks for text with Ok/Cancel buttons that close the DialogHost. That, however, is done from the View. If you need to close it from the ViewModel...

Honestly I'm not sure what would be the best solution. The Close action must be handled by the DialogHost object.

Could MvvmDialogs support `ICloseable` interface for DialogHost dialogs? Perhaps...

**Comment by mysteryx93** [comment]
Fixed in v2.0-rc1
https://github.com/mysteryx93/HanumanInstitute.MvvmDialogs/releases/tag/v2.0-rc1
